libs/templates/404/404.js

```diff
@@ -1,13 +1,28 @@
-import { getConfig, createTag } from '../../utils/utils.js';
+import { getConfig, createTag, loadArea, getMetadata } from '../../utils/utils.js';
 import { sampleRUM } from '../../utils/samplerum.js';
 
-async function load404() {
-  const config = getConfig();
+const { base, contentRoot, locale } = getConfig();
+
+async function get404(path) {
+  const { prefix } = locale;
+  const href = path || `${base}${prefix}/fragments/404#_dnt`;
+  const para = createTag('p', {}, createTag('a', { href }, href));
+  const section = createTag('div', null, para);
+
   const main = document.body.querySelector('main');
+  main.append(section);
+  await loadArea(main);
+}
+
+async function getLegacy404() {
+  const { body } = document;
+  body.classList.remove('404');
+  body.classList.add('legacy-404');
+  const main = body.querySelector('main');
   main.innerHTML = '';
-  let resp = await fetch(`${config.locale.contentRoot}/404.plain.html`);
+  let resp = await fetch(`${locale.contentRoot}/404.plain.html`);
   if (!resp || !resp.ok) {
-    const root = config.contentRoot || '';
+    const root = contentRoot || '';
     resp = await fetch(`${root}/404.plain.html`);
```

Comment:
@aishwaryamathuria Can you address this check warning?

Reply:
@rgclayton I have updated the unit test to cover teh if condition